docs/artifactory/installtransfer/help.go

```diff
@@ -0,0 +1,12 @@
+package installtransfer
+
+var Usage = []string{"rt transfer-install <server-id> [command options]"}
+
+func GetDescription() string {
+	return "Download and install the data-transfer user plugin."
```

Comment:
Let's add that the user must run this command in the source Artifactory machine

Reply:
done